packages/jaeger-ui/src/components/Monitor/ServicesView/operationDetailsTable/opsGraph.tsx

```diff
@@ -60,25 +55,22 @@ export class OperationsGraph extends React.PureComponent<TProps> {
 
     return (
       <div className="ops-container">
-        <XYPlot
-          margin={{
-            left: 0,
-            right: 0,
-            bottom: 1,
-            top: 2,
-          }}
-          width={100}
-          height={15}
-        >
-          <AreaSeries
-            className="area-series-example ops-graph-style"
-            curve="curveLinear"
-            color={color}
+        <ResponsiveContainer width={100} height={15}>
+          <ComposedChart
+            margin={{
+              left: 0,
+              right: 0,
+              bottom: 1,
+              top: 2,
+            }}
             data={dataPoints}
-            {...dynProps}
-          />
-          <LineSeries className="area-elevated-line-series" color={color} data={dataPoints} {...dynProps} />
-        </XYPlot>
+          >
+            <XAxis dataKey="x" hide />
+            <YAxis hide />
```

Comment:
Why are we using `hide` attribute here? Can't these components be just omitted?

Reply:
Yes, we can omit these components.